Tests/UITests/TestApp/TestApp.swift

```diff
@@ -12,7 +12,7 @@ import SwiftUI
 
 @main
 struct UITestsApp: App {
-    @UIApplicationDelegateAdaptor(TestAppDelegate.self) var appDelegate
+    @UIApplicationDelegateAdaptor(TestAppDelegate.self) var appDelegate // swiftlint:disable:this attributes
```

Comment:
Similar to https://github.com/StanfordSpezi/SpeziHealthKit/pull/7, we can disable the rule here in this PR.